src/components/StrategyTradesTable/StrategyTradesTable.js

```diff
@@ -28,6 +28,7 @@ const { getCurrencySymbolMemo } = reduxSelectors
 
 const StrategyTradesTable = ({
   results,
+  metrics,
```

Comment:
I think we should not export metrics on this export click as there is separate export action to export performance metrics. This one should only export positions and trades.